src/azure-cli/azure/cli/command_modules/profile/tests/latest/test_auth_e2e.py

```diff
@@ -80,3 +82,51 @@ def test_conditional_access_mfa(self):
 
         self.cmd('logout')
         # endregion
+
+
+class CAEScenarioTest(LiveScenarioTest):
+
+    def setUp(self):
+        super().setUp()
+        # Clear MSAL cache to avoid unexpected tokens from cache
+        self.cmd('az account clear')
+
+    def tearDown(self):
+        self.cmd('az account clear')
+
+    def _retry_until_error(self, cmd):
+        remaining_reties = ARM_MAX_RETRY
+        while remaining_reties > 0:
+            remaining_reties -= 1
+            sleep(ARM_RETRY_INTERVAL)
+            self.cmd(cmd)
+        raise AssertionError("Retry chance exhausted.")
+
+    def test_client_capabilities(self):
+        # Verify the access token has CAE enabled
+        result = self.cmd('account get-access-token').get_output_in_json()
+        access_token = result['accessToken']
+        decoded = decode_access_token(access_token)
+        self.assertEqual(decoded['xms_cc'], ['CP1'])  # xms_cc: extension microsoft client capabilities
```

Comment:
It seems `xms_ssm` claims is dropped from GA version of CAE, so this test is no longer needed:

```py
self.assertEqual(decoded['xms_ssm'], '1')  # xms_ssm: extension microsoft smart session management
```